src/components/Profile/MyPosts/MyPosts.js

```diff
@@ -13,7 +13,14 @@ const MyPosts = (props) => {
     newPostText,
   } = props;
 
-  const postsElements = posts.map((p) => <Post message={p.message} likesCount={p.likesCount} />);
+  const postsElements = posts.map((p) => (
```

Comment:
First of all, don't use the abbreviations or short variable names - there're hard to read.
Secondly, consider using destructuring here.

Reply:
/fxd